### ISSUE TITLE
`collapse-footer`: Option to click to expand/collapse

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6833]
### Why this would be helpful

Sometimes the user can accidentally hover over the footer area, causing it to expand unexpectedly, which can be jarring.

It can also hinder touch screen compatibility to rely on hovering to hide or show a visual element.

### My suggestion

A solution for users who are getting tired of this might be to add an option to change how you expand the footer. The setting could be called "expand on" with the options "hover" and "click". When "click" is selected, the footer would be able to be expanded by clicking it and collapsed by clicking outside of it, instead of having the state change when the mouse enters or leaves it.

### Possible alternatives

Require the mouse to be hovering over the footer area for a moment before it expands.

Touches could also be accounted for in tandem with hover detection.

**Comment by DNin01** [comment]
Should touch input testing be needed and a tester doesn't have access to a touch screen, Chrome's developer tools feature a device emulation tool, which should let us simulate touch input.
